docs/source/index.rst

```diff
@@ -57,10 +63,18 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    torch.utils.dlpack <dlpack>
    torch.utils.model_zoo <model_zoo>
    torch.utils.tensorboard <tensorboard>
-   type_info
+   torch.__config__ <__config__>
+   rpc
```

Comment:
Maybe rpc should go closer to `torch.distributed`? The package structure is `torch.distributed.rpc`.

Reply:
Again, this change goes back to the initial idea to group bigger topics (e.g. Quantization, Distributed RPC) together because the header for rpc renders as "Distributed 
RPC"; however, the more I think about it I wonder if we have 2 sections - one with all the doc strings alphabetized all using the torch.something naming schema (so for Quantization and Distributed RPC we would pull the docstring sections out of those docs) and a second section that includes these bigger topics.  In this paradigm, we'd remove the Recents/Popular section.

Reply:
> Again, this change goes back to the initial idea to group bigger topics (e.g. Quantization, Distributed RPC) together because the header for rpc renders as "Distributed
RPC"

Hmm I see. I'm fine with performing this reordering for RFC right now because it wasn't grouped with torch.distributed before. In the long term I think it would be nice to think about how the RPC things fit together with torch.distributed.